modelling/src/main/java/org/axonframework/modelling/command/StatefulCommandHandler.java

```diff
@@ -33,11 +33,11 @@
  *
  * @author Mitchell Herrijgers
  * @see StateManager
- * @see StatefulCommandHandlerRegistry
+ * @see org.axonframework.commandhandling.CommandHandlerRegistry
  * @since 5.0.0
  */
 @FunctionalInterface
-public interface StatefulCommandHandler extends MessageHandler {
+public interface StatefulCommandHandler extends CommandHandler {
```

Comment:
Ow interesting, I assumed this might have been thrown away as well. Then again, the interface itself still has value...you can just, as shown, default its implementation, making it a lot more straightforward to subscribe them to a `CommandHandlerRegistry`.

Reply:
I've added it to the discussion list for tomorrow. Curious to hear what the rest thinks :-)
Or, you can nudge everybody right away to get an opinion! ;-)

Reply:
@zambrovski @hjohn  @nordicthings @CodeDrivenMitch